src/validation.h

```diff
@@ -533,10 +533,11 @@ class CChainState
     arith_uint256 nLastPreciousChainwork = 0;
 
     /**
-     * the ChainState CriticalSection
-     * A lock that must be held when modifying this ChainState - held in ActivateBestChain()
+     * The ChainState Mutex
+     * A lock that must be held when modifying this ChainState - held in ActivateBestChain() and
+     * InvalidateBlock()
```

Comment:
nit (please ignore)

With new annotations some parts of the comment look redundant:
```suggestion
     * A lock that must be held when modifying this ChainState
```